package.json

```diff
@@ -2,14 +2,24 @@
   "name": "redux-query-sync",
   "description": "Synchronise URL query parameters and redux state.",
   "author": "Gerben <gerben@treora.com>",
-  "keywords": ["redux", "sync", "url", "location", "query", "search", "parameters"],
+  "keywords": [
+    "redux",
+    "sync",
+    "url",
+    "location",
+    "query",
+    "search",
+    "parameters"
+  ],
```

Comment:
Styling changes would also better be done separately. One PR for one fix. Feel free to make a separate PR with style changes if you wish!